solr/core/src/java/org/apache/solr/api/V2HttpCall.java

```diff
@@ -360,6 +366,85 @@ protected void execute(SolrQueryResponse rsp) {
     SolrCore.postDecorateResponse(handler, solrReq, rsp);
   }
 
+  @Override
+  protected void populateTracingSpan(Span span) {
+    // Set db.instance
+    String coreOrColName = this.origCorename;
+    if (coreOrColName == null) {
+      Map<String, String> pathTemplateValues = getUrlParts(); // == solrReq.getPathTemplateValues()
+      coreOrColName = pathTemplateValues.get("collection");
+      if (coreOrColName == null) {
+        coreOrColName = pathTemplateValues.get("core");
+      }
+    }
+    if (coreOrColName != null) {
+      span.setTag(Tags.DB_INSTANCE, coreOrColName);
+    }
+
+    // Get the templatize-ed path
+    String path;
+    if (api instanceof AnnotatedApi) {
+      // ideal scenario; eventually everything might be AnnotatedApi?
+      var aapi = (AnnotatedApi) api;
+      path = aapi.getEndPoint().path()[0]; // consider first to be primary
+    } else {
+      path = computeEndpointPath();
+      // TODO consider getValidators, looking for command & path?
```

Comment:
this is correct @dsmiley

Reply:
@noblepaul I'm looking at it now and I don't see how getValidators() can tell me what the template path is for the current request (the equivalent of `EndPoint.path()`).  Can you advise?

The `computeEndpointPath() algorithm seems pretty reasonable to me, and not slow.

Reply:
` Api.getSpec()._get("url", "paths")[0]`

Reply:
Thanks!
I tried this:
```
      @SuppressWarnings("unchecked")
      List<String> paths = (List<String>) api.getSpec()._get(List.of("url", "paths"), null);
      if (paths != null && !paths.isEmpty()) {
        // TODO use "paths[0]" above but would rather see a faster implementation first
        path = paths.get(0);
      }
```
And it worked for some endpoints but not for collection/core handlers which falsely returned "/update" instead of "/collections/{collection}/update".  And when /json was tack'ed on, it wasn't reflected there either but I suppose that could be minor.

So I'll stick with the current code (which you approved today).